content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,49 @@ menu:
 weight: 101
 ---
 
+## v2.0.8 [2021-08-09]
+
+### Breaking change: Linux package updates for influx CLI
+
+If you install InfluxDB using `dpkg -i`, now you must download and install both `influxdb2-2.0.8-amd64.deb` (InfluxDB) and `influxdb2-client-2.0.8-amd64.deb` (influx CLI) (or the RPM equivalent) to install the `influx` CLI (previously, the CLI was packaged with InfluxDB).
+
+This update was made to support the standalone installation of the `influx` CLI for InfluxDB Cloud users. Linux packages (`.deb` and `.rpm`) were modified as follows:
+
+- Add new `influxdb2-cli` package that contains only the `influx` CLI binary.
+- Remove the `influx` CLI binary from the `influxdb2` package. Now, the `influxdb2` package declares a recommended dependency on the `influxdb2-cli` package. **No breaking change** if you install using `apt install` or `apt upgrade`, as `apt` fetches and installs the recommended dependency.
+
```

Comment:
Thank you, @danxmoran, please lmk if changes above are accurate. 

@pierwill, will you double-check to verify verbiage is consistent with our usual installation terminology?

@russorat, are you OK with mentioning this in release notes only for 2.0.8? We planned to roll the CLI doc separation updates in with 2.1, but lmk your thoughts on whether we need to make specific changes to install docs for this release.